server/testdriver/core/src/main/java/org/infinispan/server/test/core/ForkedInfinispanServerDriver.java

```diff
@@ -37,17 +44,34 @@ public void prepare(String name) {
 
    @Override
    protected void start(String name, File rootDir, File configurationFile) {
-      String allServerHomes = System.getProperty(TestSystemPropertyNames.INFINISPAN_SERVER_HOME);
-      if (allServerHomes == null) {
-         throw new IllegalArgumentException("You must specify a " + TestSystemPropertyNames.INFINISPAN_SERVER_HOME + " property pointing to a comma-separated list of server homes.");
+      if (configuration.isParallelStartup()) {
+         throw new IllegalStateException("Forked mode doesn't support parallel startup");
       }
-      String[] serverHomes = allServerHomes.replaceAll("\\s+", "").split(",");
-      if (serverHomes.length != configuration.numServers()) {
+      String globalServerHome = System.getProperty(TestSystemPropertyNames.INFINISPAN_TEST_SERVER_DIR);
```

Comment:
This appears wrong, please see the fix in PR #8378.

Reply:
We need to wait for #8378 and then merge this.

Reply:
You can resolve this topic (and still needs a rebase).